### PR TITLE
Clean up default.html layout

### PR DESCRIPTION
**Pull request by codehag**
- move header into its own file
- move menu to its own file
- change css a bit to reflect future design

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -1,125 +1,21 @@
 <!DOCTYPE html>
-<html lang="{{ site.lang | default: "en-US" }}">
```
Comment (codehag) [inline _layouts/default.html]: whoops

**Review by xtuc — APPROVED** [review]
looks good, thanks.

Also we could setup netlify for a preview on PRs?

**Review by jasonwilliams — APPROVED** [review]

**Comment by codehag** [comment]
@xtuc sure, lets do that

**Comment by xtuc** [comment]
@codehag could you do it? I don't have the access. You can ping me if Netlify is not free.

**Pull request merged**